packages/Textarea/src/Textarea.styles.js

```diff
@@ -0,0 +1,83 @@
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const textareaStyles = () => `
+  position: relative;
+
+  .form-textarea__textarea {
+    ${stylers.placeholders};
+
+    background-color: ${tokens.color.white};
+    border: 1px solid ${tokens.border.color};
+    border-radius: ${tokens.border.radius};
+    box-shadow: none;
+    box-sizing: border-box;
+    color: ${tokens.color.black};
+    display: block;
+    margin: 0;
+    min-height: 80px;
+    padding: ${tokens.space};
+    resize: none;
+    width: 100%;
+
+    &:hover:not(.is-disabled):not(.is-readonly):not([disabled]) {
+      border-color: ${tokens.color.blackLighten30};
+    }
+
+    &.form-element--has-error {
+      border-color: ${tokens.color.orange};
+    }
+
+    &:focus {
+      background-color: ${tokens.color.white};
+      border-color: ${tokens.highlight.active.noBorder.borderColor};
+      box-shadow: ${tokens.highlight.active.noBorder.boxShadow};
+      outline: none;
+    }
+  }
+
+  .form-textarea__aria-label {
+    ${stylers.visuallyHidden}
+  }
+
+  // Sizes
+
+  &.form-textarea--small .form-textarea__textarea {
+    ${stylers.fontSize(-2)};
+  }
+
+  &.form-textarea--medium .form-textarea__textarea {
+    ${stylers.fontSize(-1)};
+  }
+
+  &.form-textarea--large .form-textarea__textarea {
+    ${stylers.fontSize()};
+  }
+
+  // Read Only
+
+  &.form-textarea--is-readonly .form-textarea__textarea {
+    ${stylers.readOnlyFormStyles};
+    cursor: text;
+  }
+
+  // Disabled
+
+  &.form-textarea--is-disabled, &[disabled] {
+    textarea.form-textarea__textarea {
+      ${stylers.disabledFormStyles}
```

Comment:
I just noticed that I _think_ there's something wrong with this `include`... can you check with a designer if the `isDisabled` state should have a grey background? And double check the grey background style for the `isReadOnly` state?

Reply:
@mikrotron , talked with Sarah. she's okay with it, except that the disabled text color probably doesn't pass contrast. But for now we can leave it as is until they change it